articles/overview/how-to-learn-quantum-computing.md

```diff
@@ -28,11 +28,12 @@ These concepts will give you a good introduction to the fundamental knowledge yo
 
 ## Code first, then learn! Or learn more, then code!
 
-Now that you have the basics, you're ready to start writing your first quantum programs.  Or maybe you want to learn more before coding.  Either way, our [Getting Started with the Quantum Development Kit](xref:microsoft.quantum.welcome) provides a series of steps in learning, which includes:
+Now that you have the basics, you're ready to start writing your first quantum programs.  Or maybe you want to learn more about Q# before coding.  Either way, our [Getting Started with the Quantum Development Kit](xref:microsoft.quantum.welcome) provides a series of steps in learning, which includes:
+
+* A tutorial for [writing your first Q# quantum program](xref:microsoft.quantum.write-program) and learning more about the basics of Q#.
+* A [Getting Started guide to Grover's Search](xref:microsoft.quantum.quickstarts.search), a Q# program (Grover’s search algorithm) that demonstrates some of the advantages of programming quantum solutions with a high-level Q# quantum programming language to express quantum algorithms.
+* An introduction to the Q# libraries, our Quantum Katas, and other tools of the QDK that provide you with what you need to develop quantum programs for today’s greatest challenges.
```

Comment:
I admit it is circular, but I don't think we should repeat the same list in two places.  The getting started page is the landing page from our QDK website.  So that's the place I think we should centralize the learning journeys.  This article should talk about different learning paths at a high level and then link back to the getting started page.  I'll take another stab at this article.